apps/whale-api/src/e2e.module.ts

```diff
@@ -59,7 +61,13 @@ export async function stopTestingApp (container: MasterNodeRegTestContainer | Te
 
 async function createTestingModule (url: string): Promise<TestingModule> {
   return await Test.createTestingModule({
-    imports: [AppModule.forRoot('memory')]
+    imports: [
+      CacheModule.register({ max: 10_000 }),
+      AppModule.forRoot('memory')
+    ],
+    providers: [
+      DeFiDCache
+    ]
```

Comment:
```suggestion
    imports: [AppModule.forRoot('memory')]
```

Don't think it's necessary to register CAcheModule and DeFiDCache here, since they're already registered in module.api/_module which is a part of AppModule